src/resource_server/utils/cmd.py

```diff
@@ -64,7 +67,7 @@ def load_template_values(cmd_config: Dict[str, any], target: str) -> Dict[str, a
         # Convert the Dict into a json template
         template = json.dumps(endpoint)
         # merge both parameter where local_param will replace global params
-        command = json.loads(Template(template).render({** parameters, **local_param}))
+        command = json.loads(Template(template).render({** parameters, **local_param, **request_params}))
```

Comment:
This cloud be problematic .... 
It makes it possible to override all template variables with request_params.

Reply:
isn't something that we would like to do?

Reply:
the way you do it here I can change almost any aspect of the remote invocation ...
I can even let it call a different ssh server wherever on the internet (or within our infrastructure)
I can put my custom scripts into my home folder and invoke them via some endpoint here that allows me to override the bit I want accordingly ....

Reply:
btw... this is a serious one ... with potential to be exploited

Reply:
Demo: 
```
import json
from jinja2 import Template
x = {"test": "{{ var }}"}
vars = {"var": '", "test": 1, "dummy": "'}
json.loads(Template(json.dumps(x)).render(**vars))
```
generates:
```
{'test': 1, 'dummy': ''}
```